portfolio/src/main/webapp/index.html

```diff
@@ -4,21 +4,24 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBIu0oi9bL0KdgagK9JbHg1kpmbDXglptk"></script>
```

Comment:
Is it a public key that is safe to expose on github? Just to clarify :)

Reply:
In the walkthrough it was written this: The Google Maps library requires an API key, which is a special string that
tells the library which project is making the request. So before you can start
writing code, you need to obtain an API key.

Follow
[these steps](https://developers.google.com/maps/documentation/javascript/get-api-key)
to get an API key.

And I generated it for my project. What should I do?